curl-install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Exit on error, run verbose
+set -ev
+
+# Run from a script to make sure Travis CI handles failures properly:
+# http://steven.casagrande.io/articles/travis-ci-and-if-statements/
+
+if   [ $TRAVIS_PYTHON_VERSION == 2.6 ]; then curl https://bootstrap.pypa.io/2.6/get-pip.py | python;
```

Comment:
shell scripts don't require semi-colons in the the end of lines: `\n` is already a separator itself.